chains/ethereum/connection_test.go

```diff
@@ -16,15 +16,15 @@ import (
 func TestConnect(t *testing.T) {
 	client := ethtest.NewClient(t, TestEndpoint, AliceKp)
 
-	_ = deployTestContracts(t, client, aliceTestConfig.id, AliceKp)
+	_ = deployTestContracts(t, client, aliceTestConfig.Id, AliceKp)
```

Comment:
Since the no connection no longer lives here perhaps the tests should move too

Reply:
The main reasoning for not moving the test is that it references `test_utils_test.go` quite a bit. So either the connections/evm package would reference the ethereum package, which seems wrong, or we would have to redefine a lot of variables, which violate Don't Repeat Yourself.